pkg/meta/redis.go

```
@@ -3283,11 +3341,25 @@ func (m *redisMeta) LoadMeta(r io.Reader) error {
 
 	counters := &DumpedCounters{}
 	refs := make(map[string]int)
+
+	maxNum := 10000
+	pool := make(chan struct{}, maxNum)
+	var g errgroup.Group
 	for _, entry := range entries {
-		if err = m.loadEntry(entry, counters, refs); err != nil {
-			return err
-		}
+		tmpE := entry
```

we should fail fast if err is not nil